src/Makemodule.am

```diff
@@ -111,28 +114,30 @@ src_libczmq_la_LDFLAGS += \
     -avoid-version
 endif
 
-src_libczmq_la_LIBADD = \
-    ${zmq_LIBS}
-
-if ON_ANDROID
-src_libczmq_la_LIBADD += \
-    -llog
```

Comment:
I noticed this link to `-llog` on android got clobbered by zproject.  We need this to log to system service on android, but as far as I know, zproject does not have an abstraction that allows us to add this linked library only on this platform.  @hintjens, any thoughts?